test/include/test.hpp

```diff
@@ -606,6 +631,10 @@ struct driver
             {
             }
         }
+        auto finish = std::chrono::steady_clock::now();
+        auto elapsed_ms =
+            std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(finish - start)
```

Comment:
Milliseconds may be too coarse sometimes. Should we allow (as a configurable option, if that is a possibility) a higher granularity here..? Thanks.

Reply:
Milliseconds is a good default as it can handle fast times and slow times without getting to large for most cases. 

We can add a flag to change the unit in the future after we have an idea of which type of units we would like to use.

Reply:
Worth adding as a TODO?

Reply:
[ external commentary] The duration representation is `double` - you will get fractional milliseconds https://godbolt.org/z/8e5zY3qW7.

Reply:
Thanks @sean-parent . Good catch, and the URL gives a perfect illustration.